src/api/app/views/layouts/webui/responsive_ux/_logged_navigation.html.haml

```diff
@@ -0,0 +1,27 @@
+.navbar-collapse.offcanvas-collapse.navbar-dark
+  .navbar-nav
+    .nav.justify-content-end.py-2
+      %button.navbar-toggler{ 'type': 'button', data: { toggle: 'offcanvas' }, aria: { expanded: 'false', label: 'Toggle navigation' } }
+        %i.fas.fa-times
+  %ul.navbar-nav.ml-auto.pt-0.text-nowrap
+    %li.nav-item
+      = link_to(user_path(User.session!), id: 'link-to-user-home', class: 'nav-link') do
+        = User.session!.login
```

Comment:
```suggestion
        = image_tag_for(User.session, size: 40, custom_class: 'rounded-circle bg-light')
        Your Profile
```

Reply:
Done! Now it looks like:
<img width="200" src="https://user-images.githubusercontent.com/2581944/73750365-5746a680-475d-11ea-99d8-7f977f6e58fb.png">